src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -403,7 +402,7 @@ private static boolean checkForLatestEngineVersionOut() {
           .isLessThan(latestEngineOut.getLatestVersionOut())) {
         SwingUtilities
             .invokeLater(() -> EventThreadJOptionPane.showMessageDialog(null, latestEngineOut.getOutOfDateComponent(),
-                "Please Update TripleA", JOptionPane.INFORMATION_MESSAGE, false, new CountDownLatchHandler(true)));
+                "Please Update TripleA", JOptionPane.INFORMATION_MESSAGE));
```

Comment:
This change will cause a conflict with #2284, but the resolution should be pretty easy since this code is slated for deletion.

Reply:
I'm not even sure it actually will...
If I recall correctly, this line isn't affected

Reply:
Why is the boolean param being dropped here?

Reply:
Derp.  Yes, of course, you're correct.  I saw "checkForLatestEngineVersion" and somehow read it as having to do with "areWeOldExtraJar".  Next time I need to actually look at line numbers. :smile:

Reply:
Well it does indirectly ^^
The latest version check is not being executed when the game is running using an old jar.

Reply:
> Why is the boolean param being dropped here?

Good question, but, technically, it's not.  I changed this line from calling `showMessageDialog(Component, Object, String, int, boolean, CountDownLatchHandler)` to call `showMessageDialog(Component, Object, String, int)`.  The latter calls `showMessageDialog(Component, Object, String, int, CountDownLatchHandler)` with a final argument equivalent to `new CountDownLatchHandler(true)`.  Then that overload calls the original method with `useJLabel` equal to `false`.  Thus, it's equivalent to the original call (this is more easily seen if you navigate the calls with <kbd>F3</kbd> in Eclipse).

Changing which overload is called was intentional because the value of both `useJLabel` and `latchHandler` are incidental to this particular call site and can be hidden from the reader.  The only other places where the overload that takes `useJLabel` is called all pass `true` for this parameter.  `false` is effectively a default value.

Reply:
Ah, of course. 
I was so focused on this boolean param, that I completely forgot it was false ^^

Reply:
> The only other places where the overload that takes useJLabel is called all pass true for this parameter. false is effectively a default value.

What about renaming the method instead of overloading it then?

Reply:
:+1: I will make the overload in question private and add a new public method with a hopefully more descriptive name.